src/java/org/apache/cassandra/tools/nodetool/CMSAdmin.java

```diff
@@ -269,4 +271,76 @@ public void execute(NodeProbe probe)
             probe.getCMSOperationsProxy().resumeDropAccordTable(tableId);
         }
     }
+
+    @Command(name = "dumpclustermetadata", description = "Dumps Cluster Metadata into a file")
```

Comment:
nit: since this is already gated by CMS admin, I would just use `dump_metadata` here maybe?

Reply:
What abut just `dump` ?